packages/skeleton/src/lib/utilities/index.ts

```diff
@@ -0,0 +1,28 @@
+import { initializeModalStore } from './Modal/stores.js';
+import { initializeToastStore } from './Toast/stores.js';
+import { initializeDrawerStore } from './Drawer/stores.js';
+
+/**
+ * Used to initialize the stores for the `Modal`, `Toast`, and `Drawer` utilities.
+ *
+ * @example
+ * ```svelte
+ * <!-- App's root +layout.svelte -->
+ * <script>
+ * 	import { initializeStores, Toast, Modal, Drawer } from "@skeletonlabs/skeleton";
+ *
+ * 	initializeStores();
+ * </script>
+ *
+ * <Toast />
+ * <Modal />
+ * <Drawer />
+ * ```
+ */
+export function initializeStores() {
+	const modalStore = initializeModalStore();
+	const toastStore = initializeToastStore();
+	const drawerStore = initializeDrawerStore();
+
+	return { drawerStore, modalStore, toastStore };
```

Comment:
Should we return these here or is that just going to lead to people using it for anti patterns like prop drill down?

Reply:
@Sarenor Fair point! I figured that there was some use case where someone would want to immediately use a store after initialization, but I'd be down to just making this return void instead.

Reply:
I'm open to discussion, there might be a use for it as you said. Just want to avoid people thinking they have to pass it down

Reply:
Nah, I'm convinced. I was debating this internally anyway when I initially wrote it. Having a single interface for retrieving the stores is better and more clear to the users!